addon/plugins/table-of-contents-plugin/utils/index.ts

```diff
@@ -0,0 +1,43 @@
+type OutlineEntry = {
+  content: string;
+  pos: number;
+  children?: OutlineEntry[];
+};
+
+type LI = (
```

Comment:
this type is actually just https://prosemirror.net/docs/ref/#model.DOMOutputSpec so you can import that
but not a bad try :)